### PR TITLE
Admin Info: Modify Uptime to return seconds

### PR DESCRIPTION
**Pull request by kannappanr**
## Description
Convert Uptime to return seconds instead of time.Duration

## Motivation and Context
Easier to see instead of time.Duration

## How to test this PR?
use the madmin example file and look at the uptime field's value

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
- [ ] Fixes a regression (If yes, please add `commit-id` or `PR #` here)
- [ ] Documentation needed
- [ ] Unit tests needed
- [ ] Functional tests needed (If yes, add [mint](https://github.com/minio/mint) PR # here: )

### CODE REVIEW

**Review by ebozduman — APPROVED** [review]
LGTM

**Review by krishnasrinivas — APPROVED** [review]

File: cmd/admin-server-info.go [hunk 1]
```diff
@@ -242,7 +243,7 @@ func getLocalServerProperty(endpointZones EndpointZones, r *http.Request) madmin
 	return madmin.ServerProperties{
 		State:    "ok",
 		Endpoint: addr,
-		Uptime:   UTCNow().Sub(globalBootTime),
+		Uptime:   uint64(UTCNow().Sub(globalBootTime) / time.Second),
```
Comment (harshavardhana) [inline cmd/admin-server-info.go]: If you are sending uint64 then you should send UTCNow().Unix()-globalBootTime.Unix()

**Review by harshavardhana — CHANGES_REQUESTED** [review]

**Comment by minio-ops** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-compression-fs.sh | :heavy_check_mark: |
| mint-worm.sh | :heavy_check_mark: |
| mint-fs.sh | :heavy_check_mark: |
| mint-gateway-nas.sh | :heavy_check_mark: |
| mint-compression-xl.sh | :x: [more...](#bf8514ae-924e-46a0-b23f-3571dfb5e3e5) |
| mint-xl.sh | :x: [more...](#b3f39e0d-5668-4f82-b359-9b69152ca903) |
| mint-large-bucket.sh | :x: [more...](#d1508e42-3faf-4df5-847c-741796dc0be9) |
| mint-compression-dist-xl.sh | :x: [more...](#bd768c2d-84e3-4d61-9c23-1bee83b6737d) |
| mint-dist-xl.sh | :x: [more...](#d4949a30-7823-444e-93aa-9ab95c7d5533) |
<a name="bf8514ae-924e-46a0-b23f-3571dfb5e3e5"></a>
### 8635-28c91a9/mint-compression-xl.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.52:30369
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp 353b4a4cdf11:/mint/log /tmp/mint-logs'

(1/14) Running aws-sdk-go tests ... FAILED in 6 minutes and 5 seconds
{
  "alert": "",
  "args": {
    "bucketName": "aws-sdk-go-test-dlunacdhtb5cux",
    "expiry": 60000000000,
    "objectName": "presignedTest"
  },
  "duration": 365340,
  "error": "XMinioServerNotInitialized: Server not initialized, please try again.\n\tstatus code: 503, request id: 15DF76F55BA7AB0F, host id: ",
  "function": "PresignedPut",
  "message": "AWS SDK Go CreateBucket Failed",
  "name": "aws-sdk-go",
  "status": "FAIL"
}

Executed 0 out of 14 tests successfully.
```
<a name="d1508e42-3faf-4df5-847c-741796dc0be9"></a>
### 8635-28c91a9/mint-large-bucket.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.55:32083
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp 31afc87e573e:/mint/log /tmp/mint-logs'

(1/14) Running aws-sdk-go tests ... done in 1 seconds
(2/14) Running aws-sdk-java tests ... done in 1 seconds
(3/14) Running aws-sdk-php tests ... done in 53 seconds
(4/14) Running aws-sdk-ruby tests ... done in 5 seconds
(5/14) Running awscli tests ... done in 2 minutes and 12 seconds
(6/14) Running healthcheck tests ... done in 0 seconds
(7/14) Running mc tests ... done in 1 minutes and 10 seconds
(8/14) Running minio-dotnet tests ... done in 1 minutes and 12 seconds
(9/14) Running minio-go tests ... FAILED in 1 minutes and 20 seconds
{
  "args": {
    "bucketName": "minio-go-test-g4cdkja9awzn0i9h"
  },
  "duration": 40456,
  "error": "Please reduce your request",
  "function": "RemoveObjects(bucketName, objectsCh)",
  "message": "Unexpected error",
  "name": "minio-go: testRemoveMultipleObjects",
  "status": "FAIL"
}

Executed 8 out of 14 tests successfully.
```
<a name="d4949a30-7823-444e-93aa-9ab95c7d5533"></a>
### 8635-28c91a9/mint-dist-xl.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.61:32337
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp 779426c820cb:/mint/log /tmp/mint-logs'

(1/14) Running aws-sdk-go tests ... done in 0 seconds
(2/14) Running aws-sdk-java tests ... done in 1 seconds
(3/14) Running aws-sdk-php tests ... done in 45 seconds
(4/14) Running aws-sdk-ruby tests ... done in 3 seconds
(5/14) Running awscli tests ... done in 1 minutes and 59 seconds
(6/14) Running healthcheck tests ... done in 0 seconds
(7/14) Running mc tests ... done in 54 seconds
(8/14) Running minio-dotnet tests ... done in 40 seconds
(9/14) Running minio-go tests ... done in 1 minutes and 26 seconds
(10/14) Running minio-java tests ... FAILED in 18 seconds
{
  "name": "minio-java",
  "function": "listObjects(final String bucketName, final String prefix, final boolean recursive)",
  "args": "prefix :minio, recursive: true",
  "duration": 9211,
  "status": "FAIL",
  "error": "error occurred\nErrorResponse(code=SlowDown, message=Please reduce your request, bucketName=minio-java-test-13c059t, objectName=minio-java-test-15p07r1, resource=/minio-java-test-13c059t/minio-java-test-15p07r1, requestId=15DF76F566C0B5F9, hostId=aada45b1-ea3f-47b5-9c13-cebe3ad595d6)\nrequest={method=PUT, url=http://72.28.97.61:32337/minio-java-test-13c059t/minio-java-test-15p07r1, headers=Content-Type: application/octet-stream\nContent-Encoding: aws-chunked\nx-amz-decoded-content-length: 1\nHost: 72.28.97.61:32337\nUser-Agent: MinIO (amd64; amd64) minio-java/dev\nx-amz-content-sha256: STREAMING-AWS4-HMAC-SHA256-PAYLOAD\nx-amz-date: 20191212T000630Z\nAuthorization: AWS4-HMAC-SHA256 Credential=*REDACTED*/20191212/us-east-1/s3/aws4_request, SignedHeaders=content-encoding;host;x-amz-content-sha256;x-amz-date;x-amz-decoded-content-length, Signature=*REDACTED*\n}\nresponse={code=503, headers=Accept-Ranges: bytes\nContent-Length: 363\nContent-Security-Policy: block-all-mixed-content\nContent-Type: application/xml\nRetry-After: 120\nServer: MinIO/DEVELOPMENT.2019-12-11T23-41-07Z\nVary: Origin\nX-Amz-Request-Id: 15DF76F566C0B5F9\nX-Xss-Protection: 1; mode=block\nDate: Thu, 12 Dec 2019 00:06:31 GMT\n}\n >>> [io.minio.MinioClient.executeReq(MinioClient.java:1204), io.minio.MinioClient.execute(MinioClient.java:1066), io.minio.MinioClient.executePut(MinioClient.java:1430), io.minio.MinioClient.executePut(MinioClient.java:1452), io.minio.MinioClient.putObject(MinioClient.java:4511), io.minio.MinioClient.putObject(MinioClient.java:4564), io.minio.MinioClient.putObject(MinioClient.java:4478), FunctionalTest.listObject_test5(FunctionalTest.java:1485), FunctionalTest.runTests(FunctionalTest.java:3010), FunctionalTest.main(FunctionalTest.java:3148)]"
}

Executed 9 out of 14 tests successfully.
```
<a name="bd768c2d-84e3-4d61-9c23-1bee83b6737d"></a>
### 8635-28c91a9/mint-compression-dist-xl.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.58:32493
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp b68870d4159b:/mint/log /tmp/mint-logs'

(1/14) Running aws-sdk-go tests ... done in 5 seconds
(2/14) Running aws-sdk-java tests ... done in 1 seconds
(3/14) Running aws-sdk-php tests ... done in 44 seconds
(4/14) Running aws-sdk-ruby tests ... done in 3 seconds
(5/14) Running awscli tests ... done in 2 minutes and 7 seconds
(6/14) Running healthcheck tests ... done in 0 seconds
(7/14) Running mc tests ... done in 51 seconds
(8/14) Running minio-dotnet tests ... done in 38 seconds
(9/14) Running minio-go tests ... done in 1 minutes and 9 seconds
(10/14) Running minio-java tests ... FAILED in 16 seconds
{
  "name": "minio-java",
  "function": "listObjects(final String bucketName, final String prefix, final boolean recursive)",
  "args": "prefix :minio, recursive: true",
  "duration": 10251,
  "status": "FAIL",
  "error": "error occurred\nErrorResponse(code=SlowDown, message=Please reduce your request, bucketName=minio-java-test-24chpgv, objectName=minio-java-test-23u8cof, resource=/minio-java-test-24chpgv/minio-java-test-23u8cof, requestId=15DF76F2B06B8C50, hostId=7c7d9638-9f7e-48c9-89a6-973cb783295c)\nrequest={method=PUT, url=http://72.28.97.58:32493/minio-java-test-24chpgv/minio-java-test-23u8cof, headers=Content-Type: application/octet-stream\nContent-Encoding: aws-chunked\nx-amz-decoded-content-length: 1\nHost: 72.28.97.58:32493\nUser-Agent: MinIO (amd64; amd64) minio-java/dev\nx-amz-content-sha256: STREAMING-AWS4-HMAC-SHA256-PAYLOAD\nx-amz-date: 20191212T000619Z\nAuthorization: AWS4-HMAC-SHA256 Credential=*REDACTED*/20191212/us-east-1/s3/aws4_request, SignedHeaders=content-encoding;host;x-amz-content-sha256;x-amz-date;x-amz-decoded-content-length, Signature=*REDACTED*\n}\nresponse={code=503, headers=Accept-Ranges: bytes\nContent-Length: 363\nContent-Security-Policy: block-all-mixed-content\nContent-Type: application/xml\nRetry-After: 120\nServer: MinIO/DEVELOPMENT.2019-12-11T23-41-07Z\nVary: Origin\nX-Amz-Request-Id: 15DF76F2B06B8C50\nX-Xss-Protection: 1; mode=block\nDate: Thu, 12 Dec 2019 00:06:19 GMT\n}\n >>> [io.minio.MinioClient.executeReq(MinioClient.java:1204), io.minio.MinioClient.execute(MinioClient.java:1066), io.minio.MinioClient.executePut(MinioClient.java:1430), io.minio.MinioClient.executePut(MinioClient.java:1452), io.minio.MinioClient.putObject(MinioClient.java:4511), io.minio.MinioClient.putObject(MinioClient.java:4564), io.minio.MinioClient.putObject(MinioClient.java:4478), FunctionalTest.listObject_test5(FunctionalTest.java:1485), FunctionalTest.runTests(FunctionalTest.java:3010), FunctionalTest.main(FunctionalTest.java:3148)]"
}

Executed 9 out of 14 tests successfully.
```
<a name="b3f39e0d-5668-4f82-b359-9b69152ca903"></a>
### 8635-28c91a9/mint-xl.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.56:31825
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp d40fe3e2f677:/mint/log /tmp/mint-logs'

(1/14) Running aws-sdk-go tests ... done in 0 seconds
(2/14) Running aws-sdk-java tests ... done in 1 seconds
(3/14) Running aws-sdk-php tests ... done in 42 seconds
(4/14) Running aws-sdk-ruby tests ... done in 1 seconds
(5/14) Running awscli tests ... done in 57 seconds
(6/14) Running healthcheck tests ... done in 0 seconds
(7/14) Running mc tests ... done in 29 seconds
(8/14) Running minio-dotnet tests ... done in 22 seconds
(9/14) Running minio-go tests ... done in 53 seconds
(10/14) Running minio-java tests ... done in 28 seconds
(11/14) Running minio-js tests ... FAILED in 17 seconds
{
  "name": "minio-js",
  "function": "fGetObject(bucketName, objectName, filePath)",
  "args": "bucketName:minio-js-test-d8cbd675-28b1-4447-9f18-145b2e1318b1, objectName:datafile-65-MB, filePath:/tmp/datafile-65-MB.download",
  "duration": 536,
  "status": "FAIL",
  "error": "Error: Size mismatch between downloaded file and the object at /mint/run/core/minio-js/node_modules/minio/dist/main/minio.js:965:12 at nextTask (node_modules/minio/node_modules/async/dist/async.js:4578:13) at next (node_modules/minio/node_modules/async/dist/async.js:4586:13) at /mint/run/core/minio-js/node_modules/minio/node_modules/async/dist/async.js:325:20 at FSReqWrap.oncomplete (fs.js:123:15)"
}

Executed 10 out of 14 tests successfully.
```
Error: No such image: minio/minio:8635-28c91a9

File: cmd/admin-server-info.go (hunk 1 above)
Comment (kannappanr) [reply cmd/admin-server-info.go]: done

**Review by ebozduman — APPROVED** [review]
LGTM

**Comment by minio-ops** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-compression-xl.sh | :heavy_check_mark: |
| mint-xl.sh | :heavy_check_mark: |
| mint-compression-dist-xl.sh | :heavy_check_mark: |
| mint-compression-fs.sh | :heavy_check_mark: |
| mint-worm.sh | :heavy_check_mark: |
| mint-fs.sh | :heavy_check_mark: |
| mint-dist-xl.sh | :heavy_check_mark: |
| mint-gateway-nas.sh | :heavy_check_mark: |
| mint-large-bucket.sh | :x: [more...](#8d3070bf-1efe-4e3b-b509-e71854e14e43) |
<a name="8d3070bf-1efe-4e3b-b509-e71854e14e43"></a>
### 8635-b429b58/mint-large-bucket.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.54:32431
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp d4a34bea7b8f:/mint/log /tmp/mint-logs'

(1/14) Running aws-sdk-go tests ... done in 1 seconds
(2/14) Running aws-sdk-java tests ... done in 1 seconds
(3/14) Running aws-sdk-php tests ... done in 48 seconds
(4/14) Running aws-sdk-ruby tests ... done in 9 seconds
(5/14) Running awscli tests ... done in 2 minutes and 29 seconds
(6/14) Running healthcheck tests ... done in 0 seconds
(7/14) Running mc tests ... done in 1 minutes and 24 seconds
(8/14) Running minio-dotnet tests ... FAILED in 1 minutes and 18 seconds

Unhandled Exception: System.AggregateException: One or more errors occurred. (The remote server returned an error: (503) Service Unavailable.) ---> System.Net.WebException: The remote server returned an error: (503) Service Unavailable.
   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)
   at System.Threading.Tasks.TaskFactory`1.FromAsyncCoreLogic(IAsyncResult iar, Func`2 endFunction, Action`1 endAction, Task`1 promise, Boolean requiresSynchronization)
--- End of stack trace from previous location where exception was thrown ---
   at Minio.Functional.Tests.FunctionalTest.UploadObjectAsync(String url, String filePath) in /mint/run/core/minio-dotnet/FunctionalTest.cs:line 2314
   at Minio.Functional.Tests.FunctionalTest.PresignedPutObject_Test1(MinioClient minio) in /mint/run/core/minio-dotnet/FunctionalTest.cs:line 2238
   --- End of inner exception stack trace ---
   at System.Threading.Tasks.Task.Wait(Int32 millisecondsTimeout, CancellationToken cancellationToken)
   at System.Threading.Tasks.Task.Wait()
   at Minio.Functional.Tests.Program.Main(String[] args) in /mint/run/core/minio-dotnet/Program.cs:line 132

Executed 7 out of 14 tests successfully.
```
Error: No such image: minio/minio:8635-b429b58

**Review by harshavardhana — APPROVED** [review]

**Comment by minio-ops** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-compression-xl.sh | :heavy_check_mark: |
| mint-xl.sh | :heavy_check_mark: |
| mint-compression-fs.sh | :heavy_check_mark: |
| mint-worm.sh | :heavy_check_mark: |
| mint-fs.sh | :heavy_check_mark: |
| mint-gateway-nas.sh | :heavy_check_mark: |
| mint-large-bucket.sh | :x: [more...](#c8ae360c-c9d3-477c-abde-4380773da1af) |
| mint-compression-dist-xl.sh | :x: [more...](#34bce139-e1e5-4d56-a504-01bb674cf030) |
| mint-dist-xl.sh | :x: [more...](#49c4b915-03d5-47e0-8f9c-de09881999fb) |
<a name="c8ae360c-c9d3-477c-abde-4380773da1af"></a>
### 8635-6e9a654/mint-large-bucket.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.53:32504
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp 45f4d983ec9a:/mint/log /tmp/mint-logs'

(1/14) Running aws-sdk-go tests ... done in 3 seconds
(2/14) Running aws-sdk-java tests ... done in 1 seconds
(3/14) Running aws-sdk-php tests ... done in 52 seconds
(4/14) Running aws-sdk-ruby tests ... done in 19 seconds
(5/14) Running awscli tests ... done in 2 minutes and 27 seconds
(6/14) Running healthcheck tests ... done in 0 seconds
(7/14) Running mc tests ... done in 1 minutes and 6 seconds
(8/14) Running minio-dotnet tests ... done in 1 minutes and 11 seconds
(9/14) Running minio-go tests ... FAILED in 3 minutes and 18 seconds
{
  "args": {},
  "duration": 80365,
  "error": "Please reduce your request",
  "function": "testStorageClassMetadataCopyObject()",
  "message": "CopyObject failed on RRS",
  "name": "minio-go: testStorageClassMetadataCopyObject",
  "status": "FAIL"
}

Executed 8 out of 14 tests successfully.
```
<a name="34bce139-e1e5-4d56-a504-01bb674cf030"></a>
### 8635-6e9a654/mint-compression-dist-xl.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.61:32362
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp 6a96083958a5:/mint/log /tmp/mint-logs'

(1/14) Running aws-sdk-go tests ... done in 3 seconds
(2/14) Running aws-sdk-java tests ... done in 1 seconds
(3/14) Running aws-sdk-php tests ... done in 44 seconds
(4/14) Running aws-sdk-ruby tests ... done in 2 seconds
(5/14) Running awscli tests ... done in 2 minutes and 4 seconds
(6/14) Running healthcheck tests ... done in 0 seconds
(7/14) Running mc tests ... done in 54 seconds
(8/14) Running minio-dotnet tests ... done in 34 seconds
(9/14) Running minio-go tests ... done in 1 minutes and 9 seconds
(10/14) Running minio-java tests ... FAILED in 10 seconds
{
  "name": "minio-java",
  "function": "listObjects(final String bucketName, final String prefix, final boolean recursive)",
  "args": "prefix :minio, recursive: true",
  "duration": 2896,
  "status": "FAIL",
  "error": "error occurred\nErrorResponse(code=SlowDown, message=Please reduce your request, bucketName=minio-java-test-2v6otml, objectName=minio-java-test-3leec6u, resource=/minio-java-test-2v6otml/minio-java-test-3leec6u, requestId=15DF7CB853866000, hostId=eb0bdccf-b9b3-4297-87b4-6a5357aaa80b)\nrequest={method=PUT, url=http://72.28.97.61:32362/minio-java-test-2v6otml/minio-java-test-3leec6u, headers=Content-Type: application/octet-stream\nContent-Encoding: aws-chunked\nx-amz-decoded-content-length: 1\nHost: 72.28.97.61:32362\nUser-Agent: MinIO (amd64; amd64) minio-java/dev\nx-amz-content-sha256: STREAMING-AWS4-HMAC-SHA256-PAYLOAD\nx-amz-date: 20191212T015205Z\nAuthorization: AWS4-HMAC-SHA256 Credential=*REDACTED*/20191212/us-east-1/s3/aws4_request, SignedHeaders=content-encoding;host;x-amz-content-sha256;x-amz-date;x-amz-decoded-content-length, Signature=*REDACTED*\n}\nresponse={code=503, headers=Accept-Ranges: bytes\nContent-Length: 363\nContent-Security-Policy: block-all-mixed-content\nContent-Type: application/xml\nRetry-After: 120\nServer: MinIO/DEVELOPMENT.2019-12-12T01-27-20Z\nVary: Origin\nX-Amz-Request-Id: 15DF7CB853866000\nX-Xss-Protection: 1; mode=block\nDate: Thu, 12 Dec 2019 01:52:05 GMT\n}\n >>> [io.minio.MinioClient.executeReq(MinioClient.java:1204), io.minio.MinioClient.execute(MinioClient.java:1066), io.minio.MinioClient.executePut(MinioClient.java:1430), io.minio.MinioClient.executePut(MinioClient.java:1452), io.minio.MinioClient.putObject(MinioClient.java:4511), io.minio.MinioClient.putObject(MinioClient.java:4564), io.minio.MinioClient.putObject(MinioClient.java:4478), FunctionalTest.listObject_test5(FunctionalTest.java:1485), FunctionalTest.runTests(FunctionalTest.java:3010), FunctionalTest.main(FunctionalTest.java:3148)]"
}

Executed 9 out of 14 tests successfully.
```
<a name="49c4b915-03d5-47e0-8f9c-de09881999fb"></a>
### 8635-6e9a654/mint-dist-xl.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.54:31154
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp eca65323767a:/mint/log /tmp/mint-logs'

(1/14) Running aws-sdk-go tests ... done in 3 seconds
(2/14) Running aws-sdk-java tests ... done in 1 seconds
(3/14) Running aws-sdk-php tests ... done in 43 seconds
(4/14) Running aws-sdk-ruby tests ... done in 3 seconds
(5/14) Running awscli tests ... done in 2 minutes and 5 seconds
(6/14) Running healthcheck tests ... done in 0 seconds
(7/14) Running mc tests ... done in 1 minutes and 4 seconds
(8/14) Running minio-dotnet tests ... done in 36 seconds
(9/14) Running minio-go tests ... FAILED in 1 minutes and 14 seconds
{
  "args": {
    "bucket": "minio-go-testlsd239q5aq9vtglx3",
    "object": "srcObjectSSClassCopy"
  },
  "duration": 3889,
  "error": "The bucket you tried to delete is not empty",
  "function": "testStorageClassMetadataCopyObject()",
  "message": "Cleanup failed",
  "name": "minio-go: testStorageClassMetadataCopyObject",
  "status": "FAIL"
}

Executed 8 out of 14 tests successfully.
```
Error: No such image: minio/minio:8635-6e9a654

**Pull request merged**